src/swarm/client/registry/DisablableNodeSet.d

```diff
@@ -83,16 +83,14 @@ public class DisablableNodeSet : NodeSet
     ***************************************************************************/
 
     public void disable ( NodeItem node )
-    out
```

Comment:
Are these changes related to the D2 transition?

Reply:
This was mentioned in the D2-migration docs [here](https://github.com/sociomantic/backend/blob/master/doc/d2-migration/stage3.md#invariants).

Reply:
The more detailed doc about assert/enforce/verify says something different: https://github.com/sociomantic-tsunami/sociomantic/blob/master/Code/D/assert-vs-enforce.md#adapting-older-code
>All checks in in contracts should be converted to verify statements in the method/function body.
>out contracts and invariants can remain as they are, using asserts. They will not be present in release builds.

The weird thing is that we already did a sweep of removing all `in` contracts from swarm, so I'm confused as to how you've discovered some!

Reply:
Interesting.. I have found some commits that remove `in` contracts, but the ones changed in this PR were not touched back then. Maybe they were missed for some reason?

Also, the `out` contracts have been put back now.

Reply:
It _could_ be that "in contracts (containing asserts) should only be used on private methods/ functions which are called via a public API including verify/enforce checks. in contracts on non-private methods/functions should be avoided." applies to them? (https://github.com/sociomantic-tsunami/sociomantic/blob/master/Code/D/assert-vs-enforce.md#general-error-checking-principles)